README.md

```diff
@@ -1,3 +1,138 @@
+[dc:identifier](http://purl.org/dc/terms/identifier) | https://doi.org/10.5281/zenodo.3515744
+
+
+
+
+### <a name="table-sourcesindextsv"></a>Table [sources/index.tsv](./sources/index.tsv)
+
+
+CLTS is compiled from information about transcriptions and how these relate to sounds from many sources, such as phoneme inventory databases like PHOIBLE or relevant typological surveys.
+
+property | value
+ --- | ---
+[dc:extent](http://purl.org/dc/terms/extent) | 33
+
+
+
+
+#### Columns
+
+
+Name/Property | Datatype | Description
+ --- | --- | --- 
+[NAME](http://cldf.clld.org/v1.0/terms.rdf#id) | `string` | Primary key
+[DESCRIPTION](http://cldf.clld.org/v1.0/terms.rdf#description) | `string` | 
+[REFS](http://cldf.clld.org/v1.0/terms.rdf#source) | list of `string` (separated by `, `) | References [data/references.bib::BibTeX-key](./data/references.bib)
+`TYPE` | `string` | CLTS groups transcription information into three categories: Transcription systems (`ts`), transcription data (`td`) and soundclass systems (`sc`).
+`URITEMPLATE` | `string` | Several CLTS sources provide an online catalog of the graphemes they describe. If this is the case, the URI template specified in this column was used to derive the URL column in graphemes.csv.
+
+
+
+### <a name="table-datafeaturestsv"></a>Table [data/features.tsv](./data/features.tsv)
+
+
+The feature system employed by CLTS describes sounds by assigning values for certain features (constrained by sound type). The permissible values per (feature, sound type) are listed in this table.
+
+property | value
+ --- | ---
+[dc:extent](http://purl.org/dc/terms/extent) | 156
+
+
+
+
+#### Columns
+
+
+Name/Property | Datatype | Description
+ --- | --- | --- 
+[ID](http://cldf.clld.org/v1.0/terms.rdf#id) | `string` | Primary key
+`TYPE` | `string` | 
+`FEATURE` | `string` | 
+`VALUE` | `string` | 
+
+
+
+### <a name="table-datagraphemestsv"></a>Table [data/graphemes.tsv](./data/graphemes.tsv)
+
+
+property | value
+ --- | ---
+[dc:extent](http://purl.org/dc/terms/extent) | 80457
+
+
+
+
+#### Columns
+
+
+Name/Property | Datatype | Description
+ --- | --- | --- 
+`PK` | `integer` | Primary key
+`GRAPHEME` | `string` | Grapheme used in a particular transcription to denote a sound
+`NAME` | `string` | The ordered concatenation of feature values of the denoted sound<br>References [data/sounds.tsv::NAME](#table-datasoundstsv)
+`BIPA` | `string` | The grapheme for the denoted sound in the Broad IPA transcription system
+`DATASET` | `string` | Links to the source of this grapheme<br>References [sources/index.tsv::NAME](#table-sourcesindextsv)
+`FREQUENCY` | `integer` | 
+`URL` | `anyURI` | URL of the grapheme in its source online database
+`IMAGE` | `anyURI` | Image of the typeset grapheme.
+`SOUND` | `anyURI` | Audio recording of the sound being pronounced.
+`EXPLICIT` | `string` | 
```

Comment:
Name/Property | Datatype | Description
--- | --- | ---
`EXPLICIT` | `string` | Indicates whether the mapping of the sound was done manually (explicitly, `+`) or whether it was inferred from the Grapheme (`None`). 

The ALIAS is, I think, no longer in use, I do not remember when we indicated it. It would originally mean that we infer it but the original graphem is an alias, but I think this can be dropped now with our explicit mapping procedure that requires hand-crafted mappings.

Reply:
You can see that `make_dataset` no longer supports it.

Reply:
But the same could be said about `explicit`.

Reply:
While I'd say explicit is important for controlling, and also supported by `make_dataset`.